tests/platforms/test_wiscourts.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import call, patch
```

Comment:
Again, if these tests hit live sites, we should disable or delete unless they use mocks/fixtures.